ejb3/src/main/java/org/jboss/as/ejb3/component/stateless/StatelessComponentDescription.java

```diff
@@ -191,9 +191,7 @@ public void configure(DeploymentPhaseContext context, ComponentConfiguration com
             }
         });
 
-
-        if (view instanceof EJBViewDescription) {
-            EJBViewDescription ejbViewDescription = (EJBViewDescription) view;
+            EJBViewDescription ejbViewDescription = view;
             if (ejbViewDescription.getMethodIntf() == MethodIntf.REMOTE) {
```

Comment:
looks like we can directly use the variable `view`, without creating a new variable `ejbViewDescription`

Reply:
Thank you for your fast reply. I have removed the unnecessary local variable.